release-notes/con-relnotes-notable-features.adoc

```diff
@@ -3,143 +3,62 @@
 
-== Supported plugins in {product} {product-version}
-
-The following table lists the supported plugins in {product} {product-version}:
-
-[id="supported-plugins-in-rhdh"]
-.Supported plugins in {product} {product-version}
-[cols="30%,30%,10%,30%",options="header",subs="+attributes"]
-|===
-| Plugin name | Plugin ID | Version | Category
-
-| Segment | `@janus-idp/backstage-plugin-analytics-provider-segment` | 1.2.0 | Analytics
-
-|Keycloak |`@janus-idp/backstage-plugin-keycloak-backend` |1.7.6 |Identity and access management
-
-|Open Cluster Management (OCM)|`@janus-idp/Backstage-plugin-ocm`|3.5.0|Cluster management
-
-|OCM backend|`@janus-idp/backstage-plugin-ocm-backend`|3.4.6|Cluster management
-
-|Quay|`@janus-idp/backstage-plugin-quay`|1.4.6|Container management
-
-|Quay actions|`@janus-idp/backstage-scaffolder-backend-module-quay`|1.2.1|Container management
-
-|RBAC backend|`@janus-idp/backstage-plugin-rbac-backend`|1.6.3|Security
-
-|RBAC common|`@janus-idp/backstage-plugin-rbac-common`|1.1.0|Security
-
-|Tekton|`@janus-idp/backstage-plugin-tekton`|3.1.3|Cluster management
-
-|Topology|`@janus-idp/backstage-plugin-topology`|1.16.4|Cluster management
-
-|Regex actions|`@janus-idp/backstage-scaffolder-backend-module-regex`|1.2.1|Continuous deployment
-
-|ArgoCD|`@roadiehq/backstage-plugin-argo-cd`|2.3.5|Continuous deployment
-
-|ArgoCD backend|`@roadiehq/backstage-plugin-argo-cd-backend`|2.11.3|Continuous deployment
-
-|API Documentation|`@backstage/plugin-api-docs`|0.9.12|Continuous deployment
-
-|App backend|`@backstage/plugin-app-backend`|0.3.54|Continuous deployment
-
-|Auth backend|`@backstage/plugin-auth-backend`|0.19.4|Security
-
-|Auth node|`@backstage/plugin-auth-node`|0.4.0|Security
-
-|Catalog frontend|`@backstage/plugin-catalog`|1.14.0|Continuous deployment
+=== Configuration change in the GitHub Org catalog provider  
 
-|Catalog backend|`@backstage/plugin-catalog-backend`|1.14.0|Continuous deployment
+The GitHub Org catalog module for the new backend system switched from an obsolete implementation to a new one, that is, `GithubMultiOrgProvider`. This new implementation brings in changes to the default plugin configuration settings.
 
-|Catalog common|`@backstage/plugin-catalog-common`|1.0.17|Continuous deployment
+=== New Scaffolder Identity client
 
-|Catalog Graph|`@backstage/plugin-catalog-graph`|0.2.37|Continuous deployment
+Identity client functioning differs slightly between the one provided by the new backend system Scaffolder plugin, and the one used in the previous release:
 
-|Catalog Import|`@backstage/plugin-catalog-import`|0.10.1|Continuous deployment
+* The previous release uses the `DefaultIdentityClient` function without any specific option
+* New backend system Scaffolder does not use the default identity service, but still uses the deprecated scaffolder-dedicated `buildDefaultIdentityClient` function, which introduces a slight functioning change in the identity client. 
```

Comment:
Hello @shipsing , 

a) Could we be consistent? The first bullet doesn't have a period, but the second bullet does.
b) I believe, we should put a comma before **_but_** only when **_but_** is connecting two independent clauses. Here, comma before but can be dropped it seems? 

Please let me know your thoughts. Thanks!

Reply:
I have added a period for the first point. Yes, for the second point, we do not need a comma because they are not independent clauses. So, leaving as is.